src/libtrx/game/collision.c

```diff
@@ -5,17 +5,69 @@
 #include "game/lara/common.h"
 #include "game/matrix.h"
 #include "game/rooms.h"
+#include "log.h"
 #include "utils.h"
 
+static void M_FillSide(
+    const COLL_INFO *coll, COLL_SIDE *side, int32_t x_pos, int32_t z_pos,
+    int32_t y_pos, int32_t obj_height, int16_t room_num);
 static bool M_IsOnWalkable(
     const SECTOR *sector, int32_t x, int32_t y, int32_t z, int32_t room_height);
 
+static void M_FillSide(
+    const COLL_INFO *const coll, COLL_SIDE *const side, const int32_t x_pos,
+    const int32_t z_pos, const int32_t y_pos, const int32_t obj_height,
+    int16_t room_num)
+{
+    const int32_t y = y_pos - obj_height;
+    const int32_t y_top = y - 160;
```

Comment:
What does 160 signify here?

Additionally, can we put a generic comment on what this function does in very generic terms? (I know it's a nicely refactored stuff from GetCollisionInfo but I must say I don't quite understand the original function.)